### PR TITLE
Add shared_runners_enabled argument to gitlab_project resource

### PR DESCRIPTION
**Pull request by iAnomaly**

### CODE REVIEW

**Comment by iAnomaly** [comment]
@jorcau @roidelapluie, this replaces #104 with the requested missing tests. Please note that tests have been failing on master since @roidelapluie merged #132 6 days ago. This PR does not add any new failures.

Thanks!

File: gitlab/resource_gitlab_project_test.go [hunk 1]
```diff
@@ -406,7 +410,8 @@ resource "gitlab_project" "foo" {
   approvals_before_merge = 0
   wiki_enabled = false
   snippets_enabled = false
-  container_registry_enabled = false
+	container_registry_enabled = false
+	shared_runners_enabled = false
```
Comment (roidelapluie) [inline gitlab/resource_gitlab_project_test.go]: please fix the alignment

**Comment by roidelapluie** [comment]
@iAnomaly this is breaking tests

**Comment by iAnomaly** [comment]
Hey @roidelapluie. As I pointed out in my previous comment, these tests have been failing since you merged #132; 8 days before you merged my code. Specifically, commit https://github.com/terraform-providers/terraform-provider-gitlab/commit/c95dfb0b1fcac7cc4e474f59ebff585fad5209c3 is when the build stopped passing.

You can compare the test results between the two and see it is the same failure:
1. Build of https://github.com/terraform-providers/terraform-provider-gitlab/commit/c95dfb0b1fcac7cc4e474f59ebff585fad5209c3 from merge #132: https://travis-ci.org/terraform-providers/terraform-provider-gitlab/jobs/530438424
2. Build of https://github.com/terraform-providers/terraform-provider-gitlab/commit/82886a6535d353dfc31bba792d40613dbb62c977 from  merge of my pull request #134: https://travis-ci.org/terraform-providers/terraform-provider-gitlab/jobs/533726463

Specifically, the failure is from line 539 of the build log:
> --- FAIL: TestAccGitlabProject_basic (6.03s)
    testing.go:568: Step 1 error: Check failed: Check 2/2 error: 1 error occurred:
        	* Check 19/22 error: attribute tags expected "&{F:0xf04970 m:map[4151227546:tag1 1852302624:tag2] once:{m:{state:0 sema:0} done:1}}" received "&{F:0xf04970 m:map[1852302624:tag2 4151227546:tag1] once:{m:{state:0 sema:0} done:1}}"

Does that make sense? Let me know if I am misunderstanding you.

P.S. Sorry for not squashing my commits before you merged. I thought GitHub would allow you to do that automatically at merge time. Alas, I've become too comfortable with GitLab.

**Comment by roidelapluie** [comment]
Github allows me to do it but I dont like it.

I changed the value to "computed".

**Comment by roidelapluie** [comment]
(there was a failing test because of this .. in addition to the other issue)

**Pull request merged**